io/src/main/scala/sbt/io/PathMapper.scala

```diff
@@ -148,15 +148,15 @@ abstract class Mapper {
    * The resulting mappings sequence will look something like this
    *
    * {{{
-   * File($baseDirectory/extras/file1) -> "file1"
-   * File($baseDirectory/extras/file2) -> "file2"
+   * File(baseDirectory/extras/file1) -> "file1"
+   * File(baseDirectory/extras/file2) -> "file2"
    * ...
    * }}}
    *
    * @example Add a static directory "extra" and re-map the destination to a different path
    * {{{
    * mappings ++= contentOf(baseDirectory.value / "extra").map {
-   *   case (src, destination) => src -> s"new/path/$destination"
+   *   case (src, destination) => src -> s"new/path/destination"
```

Comment:
Minor thing, but the string interpolation is gone for the `destinantion` variable